requirements/constraints.txt

```
@@ -9,12 +9,12 @@
 # linking to it here is good.
 
 # Common constraints for edx repos
--c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
+#-c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
```

We need to replace this with local `common_constraints.txt` path.
```suggestion
-c common_constraints.txt
```